internal/outputgenerators/core.go

```diff
@@ -206,6 +206,10 @@ func (c Core) buildDevice(deviceId uint, exp experiment.Experiment, nodeGroup ex
 		Type:          c.deviceType(nodeGroup.NodesType),
 		Interfaces:    deviceNetworkInterfaces,
 	}
+	if nodeGroup.PredefinedPosition {
+
+		dev.Position = nodeGroup.Position
+	}
 	return dev, nil
```

Comment:
Please rather use this bracing style:
```suggestion
	}
	
	if nodeGroup.PredefinedPosition {
		dev.Position = nodeGroup.Position
	}
	
	return dev, nil
```